### ISSUE TITLE
(Wide-ResNet)Wide Residual Networks

### ISSUE THREAD

**Issue by yutayamazaki** [yutayamazaki/paper_reading#8]
### 論文
Wide Residual Networks
https://arxiv.org/abs/1605.07146

### 著者・所属機関
Sergey Zagoruyko, Nikos Komodakis
Université Paris-Est, École des Ponts
ParisTech

### 投稿日
2017/6/14

## Abstract
- Wide-ResNet

## 先行研究と比べて何がすごい？
- パラメータ数が同じ程度のResNetと比べ，精度が高く計算が数倍速い
- この時点でのSOTA

### 層の深さを減らし，幅を広げたResNet
- resnetの層はほとんど良い情報を獲得できておらず，あまり精度の向上に役立っていない可能性がある
- となるとresnetの主な利点は層を深く出来ることではなく．residual blockそれ自体
- 層を浅く幅を広くとると，16層で1000層のresnetと同様の精度を達成し，学習は2倍速かった

### ベストな構造
- Dropoutをresidual blockのConvとConvの間に入れた(conv, relu, dropout, conv, relu)
- wide-resnetではchannel数をk倍する．k=1でoriginal resnetと同じ
- residual blockをConv-BN-ReLUからBN-Conv-ReLUに変更した
- 各blockごとのConvの数は3×3がベスト
- wide-resnetの構造はdepth=28, k=10でCIFAR10のベストを出している．depth=16, k=8でもまあまあ良い

## 他
- BNはinternal covariate shift(内部共変量シフト)を削減し，層が深くパラメータ数の多いNNのoverfittingを防ぐ
- 深さや幅を広げたらそれだけ十分なregularizationを施してやる必要がある
- 同じパラメ―タ数のWide-ResNetの学習がうまく行ってるので，非常に層の深いResNetの構造がとくにregularizationの効果を持ってるわけじゃない
- 深さ方向に層を重ねるよりも，幅を取ったほうが計算的にコストが低そう

## どうやって有効だと検証した？
- 各種データセットでのSOTA

## 次に読むべき論文は？
- Net2Net: Accelerating Learning via Knowledge Transfer, https://arxiv.org/abs/1511.05641
- Identity Mappings in Deep Residual Networks, https://arxiv.org/abs/1603.05027
- Batch Normalization: Accelerating Deep Network Training by Reducing Internal Covariate Shift, https://arxiv.org/abs/1502.03167
- Network In Network, https://arxiv.org/abs/1312.4400
- Very Deep Convolutional Networks for Large-Scale Image Recognition, https://arxiv.org/abs/1409.1556
- Dropout: A Simple Way to Prevent Neural Networks from Overfitting, http://jmlr.org/papers/volume15/srivastava14a.old/srivastava14a.pdf
- Inception-v4, Inception-ResNet and the Impact of Residual Connections on Learning, https://arxiv.org/abs/1602.07261
